community/javascript/remove-unused-vars/transform.ts

```diff
@@ -0,0 +1,13 @@
+import { FileInfo } from 'jscodeshift';
+// @ts-expect-error
+import putout from 'putout';
+// @ts-expect-error
+import removeUnusedVars from '@putout/plugin-remove-unused-variables';
+
+export default function transformer(file: FileInfo) {
+  const output = putout(file.source, {
+    plugins: [removeUnusedVars],
+  });
+
+  return output.code;
+}
```

Comment:
@coderaiser You'll probably be able to spot the issue instantly, but I keep running into this:

<img width="754" alt="image" src="https://user-images.githubusercontent.com/3030010/237017631-4b7ff14d-e134-4cb1-bec6-0ca141a40ca5.png">

Reply:
Do you know of a way to preserve formatting via recast? 
<img width="498" alt="image" src="https://github.com/CodeshiftCommunity/CodeshiftCommunity/assets/3030010/c4da0df9-6a90-44f8-a5f5-e0a8c40bfbb3">

Reply:
This is how recast works when it has a little code to work with 🤷. I usually fix this with [`eslint-plugin-putout`](https://github.com/coderaiser/putout/tree/master/packages/eslint-plugin-putout#readme) which solves all formatting issues.

Anyways I’m working on [`@putout/printer`](https://github.com/putoutjs/printer) which just makes good formatting each time, not trying to preserve anything (and much easier in maintenance).